pages/linux/pvcreate.md

```diff
@@ -1,11 +1,11 @@
 # pvcreate
 
-> Initialize a disk or partition for use by LVM (Logical Volume Manager).
+> Initialize a physical volume (disk or partition) for use by the Logical Volume Manager (LVM).
 
-- Initialize `/dev/sda1` to use lvm:
+- Initialize the `/dev/sda1` device for use by LVM:
```

Comment:
The "disk or partition" comes from the [pvcreate manpage](https://linux.die.net/man/8/pvcreate):

> pvcreate initializes *PhysicalVolume* for later use by the Logical Volume Manager (LVM). Each *PhysicalVolume* can be a disk partition, whole disk, meta device, or loopback file.

Reading the diagram in [this section](https://en.wikipedia.org/wiki/Logical_Volume_Manager_(Linux)#Features), it seems like "/dev/..." are only used for logical (as opposed to physical) volumes, so I'm not really sure what to use here.

Perhaps our best bet would be to play it safe and say "the `/dev/sda1` volume", which is also consistent with the manpage's synopsis ("pvcreate [options...] PhysicalVolume"). What do you think?